### DISCUSSION TITLE
Started Discussion

### DISCUSSION THREAD

**Discussion by destroylord** [Degovan/vue-isonikah#11] [General]
Ehh rul @sahrullahh

**Comment by sahrullahh** [comment]
yaapa ya mas gaada merge ni

**Reply by destroylord** [reply]
Nah yaitu, aku gak tau ini akwoakwok
Tapi gpp wes tak pull yang master, kan cuman bikin modules wongan aku juga nyoba nyoba fetch api belum tak implement

**Reply by sahrullahh** [reply]
![This is an image](https://i.imgur.com/pGdptEk.gif)

**Reply by destroylord** [reply]
Wokeelah sek tak install laravel mix lagi, itu branch tak hapus semua bek aku

**Comment by destroylord** [comment]
Rul ada Pr tuh, coba di merge @sahrullahh

**Reply by sahrullahh** [reply]
![image](https://user-images.githubusercontent.com/49519441/147466127-84ac156b-4fb6-45b1-adf7-2e5d94762c43.png)

**Reply by sahrullahh** [reply]
pull requests nya gak ada nih

**Reply by destroylord** [reply]
Nah yaitu aku padahal sudah push make branch lain, kok pas gak bisa bikin pr gitu 😢

**Reply by destroylord** [reply]
Rul coba bikin commit baru

**Reply by destroylord** [reply]
Atau sekalian kamu bikin pr juga, bisa endak tuh @sahrullahh

**Reply by sahrullahh** [reply]
Bentar mas sek di luar 😅

**Comment by destroylord** [comment]
Udah bisa kang @sahrullahh di pr #13 udah tak coba awokaowkok

**Reply by sahrullahh** [reply]
Belum coba ni , sek nanti tak coba lagi

**Reply by destroylord** [reply]
Mantap, langsung pull aja dah yang master, pr baru tak hapus aja wes bek aku rul @sahrullahh

**Reply by sahrullahh** [reply]
bisa wes mass alhamdulilah, tapi gak bisa di pull ni masalah nya harus di clone lagi, tapi ya no problem sih

**Reply by destroylord** [reply]
Bikin modules architecturesnya belakangan aja wes rul, kebut issue yang belom-belom di isonikah tapi tetep priority project client

**Reply by sahrullahh** [reply]
ngokee siapp

**Comment by sahrullahh** [comment]
mas @destroylord  yang register tak ubah uinya soalnya gak cukup buat isi form nya

**Reply by destroylord** [reply]
Okee dahh gpp

**Comment by destroylord** [comment]
Jadi udah beres yaa rul tampilannya @sahrullahh ?

**Reply by sahrullahh** [reply]
Beres wes

**Reply by destroylord** [reply]
Wokelah, implements api ini dah yaa?? kamu mau nyoba apa gimana?? @sahrullahh

**Comment by destroylord** [comment]
Dikebut ayoo rul, tgl 20an launching MVP

**Reply by destroylord** [reply]
Cek PR ngab #16 @sahrullahh
